tests/framework/extensions/provisioning/ssh.go

```diff
@@ -0,0 +1,41 @@
+package provisioning
+
+// This file contains all tests that require to ssh into a node to run commands to check things
+// such as any stats, benchmarks, etc. For example, ssh is required to check the cpu usage of a
+// process running on an individual node.
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/pkg/nodes"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	cpuUsageVar = 100 // 100 is just a placeholder until we can determine an actual number. Even with cpu usage spiking it should not go past 100% cpu usage and previous issues concerning this were hitting around 130% and above
+	checkCPU    = "CheckCPU"
+)
+
+// This func checks the cpu usage of the cluster agent. If the usage is too high the func will return a warning.
+func CallSSHTestByName(testName string, client *rancher.Client, nodes *nodes.Node) error {
```

Comment:
We are passing in `client` as a parameter, but its not being called upon in the function, so we should remove this parameter + the associated import

Reply:
agreed